### ISSUE TITLE
Companion Object's logger is public

### ISSUE THREAD

**Issue by PCasafont** [MicroUtils/kotlin-logging#49]
Normally, when we define a logger in our class, it is private. But if we use the companion object strategy, there's no way to achieve that.

**Comment by oshai** [comment]
Yes, I personally think the file level logger is better. Do you have any suggestion how to change that?

**Comment by MigiKun** [comment]
Oh my apologies, I didn't notice your so quick response!
Why was the logger definition made public instead of protected? Can it become protected? That may help.

**Comment by PCasafont** [comment]
My apologies, I didn't notice your quuck response. What about changing it to protected? That would help, if possible.

**Comment by oshai** [comment]
If it is possible to access from the instances I guess it is fine. Can you check it? Add a PR?

**Comment by PCasafont** [comment]
I will try during the next week

**Comment by oshai** [comment]
Thanks!

**Comment by oshai** [comment]
What was the problem?

**Comment by PCasafont** [comment]
An interface cannot have protected fields
